thinkhazard/static/less/report.less

```diff
@@ -249,6 +249,10 @@ footer {
   margin-right: 10px;
 }
 
+.hazard-active {
+    font-family: "Source Sans Pro Semibold";
```

Comment:
This won't work unless this font is installed in the operating system of the user.
We load the font from `googleapis.com` so we only have the "Source Sans Pro" font-family available. See http://fonts.googleapis.com/css?family=Source+Sans+Pro

So if we want a bolder font-weight, let's use:
```
font-weight: bold;
```